README.adoc

```diff
@@ -5,9 +5,10 @@ This repository contains additional notes for the releases of SUSE Cloud Applica
 * link:/adoc/Release-Notes.adoc[Release Notes] for specific releases
 * link:/adoc/known-issues.adoc[Known issues] for known issues
 
-Some hints for special environments:
+Hints for specialized environments:
 
-* link:/adoc/Notes-CaaSP.adoc[SUSE CaaS (Container as a Service) Platform]
+* link:https://www.suse.com/documentation/suse-caasp-2/book_caasp_deployment/data/book_caasp_deployment.html[SUSE CaaS (Container as a Service) Platform & Bare Metal or Non-OpenStack Virtualization Platforms]
+* link:https://www.suse.com/documentation/suse-openstack-cloud-8/book_install/data/sec_heat_templates_install.html[SUSE CaaS (Container as a Service) Platform & SUSE OpenStack Cloud]
```

Comment:
Getting a 404 for this link

Reply:
Thanks! The book has been renamed from book_install -> book_installation at some point.